Polyhedron/doc/Polyhedron/CGAL/Polyhedron_3.h

```diff
@@ -1594,7 +1594,27 @@ class Polyhedron_3 {
   */ 
   void delegate( CGAL::Modifier_base<HDS>& m); 
 
+  
   /// @}
 
 }; /* end Polyhedron_3 */
+
+
+/*!
+\relates Polyhedron_3
+\ingroup PkgPolyhedronIOFunc
+
+*/
+template <class PolyhedronTraits_3>
+bool read_off( std::istream& in, CGAL::Polyhedron_3<PolyhedronTraits_3>& P);
+
+/*!
+\relates Polyhedron_3
+\ingroup PkgPolyhedronIOFunc
+
+*/
+template <class PolyhedronTraits_3>
+bool write_off( std::ostream& out, CGAL::Polyhedron_3<PolyhedronTraits_3>& P);
```

Comment:
I don't think we need to modify the doc of Polyhedron_3 (nor the one of Surface_mesh). The way to load/save an off is still to use operators << and >>.
I think we only need to document read_off and write_off in the FaceGraph concept.